tasks/test.rake

```diff
@@ -47,15 +47,18 @@ end
 
 rule /^test:features:.+$/ do |task|
   dir = "gems/#{task.name.split(':').last}/features"
-  tags = "-t 'not @veryslow'"
+  # Exclude support smoke tests as these require account settings
+  # the logical or with not results in skipping tests marked with
+  # @support AND @smoke only.
+  tags = "-t 'not @veryslow' -t 'not @support or not @smoke'"
```

Comment:
Is there another way to meet this condition?

Reply:
I'm not sure - this was the only way I could find to write this.

Reply:
I did test that all other smoke tests are run.  And that other support tests would be run (we don't have any.  but if we wrote them in the future they would be run.  I'm not sure if thats what we want or not.  but I was trying to make a very selective filter).